### DISCUSSION TITLE
Tensorflow intel optimization

### DISCUSSION THREAD

**Discussion by Sockolet** [nextcloud/recognize#593] [Q&A] [answered]
I'm trying to use Recognize with Tensorflow optimized for intel CPUs:
https://www.intel.com/content/www/us/en/developer/articles/guide/optimization-for-tensorflow-installation-guide.html

Is it possible? If yes, how?

Thank you very much in advance.

**Comment by marcelklehr** [comment, marked as answer]
The site you linked states

> Since TensorFlow v2.9, the oneAPI Deep Neural Network Library (oneDNN) optimizations are enabled by default.

Recognize uses libtensorflow v2.9.1
